apidoc/Titanium/App/iOS/iOS.yml

```diff
@@ -988,7 +988,10 @@ events:
 
       - name: responseText
         summary: The response text for upload tasks (since SDK 6.1.0).
-        type: String
+        type: Number
+
+      - name: statusCode
+        summary: The response status code for upload tasks (since SDK 6.1.0)
```

Comment:
I thought this the type key was fixed on the last commit. I'm adding the point now